src/helpers/string.ts

```diff
@@ -1,5 +1,9 @@
+import crypto from 'crypto';
```

Comment:
Having a dep on a crypto lib is not what we want. Just take the string length `mod 10^6`